Library/Formula/fish.rb

```diff
@@ -11,16 +14,12 @@ class Fish < Formula
 
   head do
     url "https://github.com/fish-shell/fish-shell.git", :shallow => false
-
-    depends_on "autoconf" => :build
-    # Indeed, the head build always builds documentation
-    depends_on "doxygen" => :build
   end
 
   skip_clean "share/doc"
 
   def install
-    system "autoconf" if build.head?
+    system "autoconf"
```

Comment:
Might be worth adding a small note to say that we can make this (and the autoconf) dep ` if build.head? ` again with the next big release. It's only necessary this time because they didn't cut a release for all OS versions they support, just OS X to fix the big filesystem corruption problem.